praw/models/reddit/subreddit.py

```diff
@@ -2150,3 +2168,46 @@ def revisions(self, **generator_kwargs):
         url = API_PATH['wiki_revisions'].format(subreddit=self.subreddit)
         return WikiPage._revision_generator(
             self.subreddit, url, generator_kwargs)
+
+
+class SubredditEmoji(object):
+    """Provides a set of functions to a Subreddit for emoji."""
+
+    def __getitem__(self, name):
+        """Lazily return the Emoji for the subreddit named ``name``.
+
+        This method is to be used to fetch a specific emoji url, like so:
+
+        .. code:: python
+
+           emoji = reddit.subreddit('iama').emoji['cake']
+           print(emoji)
+
+        """
+        return Emoji(self.subreddit._reddit, self.subreddit, name.lower())
+
+    def __init__(self, subreddit):
+        """Create a SubredditModeration instance.
+
+        :param subreddit: The subreddit to moderate.
+
+        """
+        self.subreddit = subreddit
+
+    def __iter__(self):
+        """Iterate through the Emoji for the subreddit.
+
+        This method is to be used to discover all emoji for a subreddit:
+
+        .. code:: python
+
+           for emoji in reddit.subreddit('iama').emoji:
+               print(emoji)
+
+        """
+        response = self.subreddit.fullname._reddit.get(
+            API_PATH['emoji_list'].format(subreddit=self.subreddit))
+        for emoji_name, emoji_data in
```

Comment:
I'm fairly certain you need a backslash (\\) after `for emoji_name, emoji_data in`, because Python doesn't like it if you continue this statement on a new line without one.